src/card/card-title.jsx

```diff
@@ -79,8 +79,11 @@ const CardTitle = React.createClass({
     const titleStyle = Object.assign({}, styles.title, this.props.titleStyle);
     const subtitleStyle = Object.assign({}, styles.subtitle, this.props.subtitleStyle);
 
+    const attrs = Object.assign({}, this.props);
+    delete attrs.title;
+
     return (
-      <div {...this.props} style={prepareStyles(rootStyle)}>
+      <div {...attrs} style={prepareStyles(rootStyle)}>
         <span style={prepareStyles(titleStyle)}>
           {this.props.title}
         </span>
```

Comment:
JSX has implemented it: https://facebook.github.io/react/docs/transferring-props.html#transferring-with-...-in-jsx